Tests/PublishTests/Tests/FileIOTests.swift

```diff
@@ -122,6 +122,19 @@ final class FileIOTests: PublishTestCase {
 
         XCTAssertEqual(itemFile?.name, "index.html")
     }
+
+    func testCleanHiddenFilesInOutputFolder() throws {
+        let folder = try Folder.createTemporary()
+        try folder.createFile(at: "Output/.hidden")
+        try folder.createFile(at: ".hidden").write("Hello, world!")
```

Comment:
I think this test would be more robust if you didn't do this, and instead just asserted that the output folder *doesn't* contain any `.hidden` file at all. Right now you're relying on the `.copyFile` command throwing an error in case a file is already present, which is something that could change in the future, and would result in this test causing false positives.

Reply:
You are right. I updated it. Thanks for review.